cmd/helm/create.go

```diff
@@ -31,7 +31,8 @@ import (
 
 const createDesc = `
 This command creates a chart directory along with the common files and
-directories used in a chart.
+directories used in a chart, just provides an e2e example and is not
```

Comment:
In general it's best to avoid using abbreviations in help text. The sentence is not grammatically correct, either. Would you mind changing this to

"... directories used in a chart. It provides a basic example and is not..."

Reply:
ok